client/state/jetpack-settings/modules/test/selectors.js

```diff
@@ -137,4 +138,19 @@ describe( 'selectors', () => {
 			expect( output ).to.be.null;
 		} );
 	} );
+
+	describe( '#isFetchingModuleList', () => {
+		it( 'should return true if the list of modules is being fetched', () => {
+			const stateIn = {
+					jetpackSettings: {
+						jetpackModules: {
+							requests: REQUESTS_FIXTURE
+						}
+					}
+				},
+				siteId = 123456;
+			const output = isFetchingModuleList( stateIn, siteId );
+			expect( output ).to.be.true;
+		} );
```

Comment:
Let's also add test cases for when the list of modules is not being fetched, and for when there is no such site at all (when it should return `null`).